tests/test_query_parameter.py

```diff
@@ -39,8 +39,8 @@ def method(
 
 
 @pytest.mark.parametrize(('query_string', 'expected_exception_message'), (
-        ('query_param=invalid_int', 'Invalid query parameter "query_param" value "invalid_int"'),
-        ('', 'Missing required query parameter "query_param"'),
+    ('query_param=invalid_int', 'Invalid query parameter "query_param" value "invalid_int"'),
```

Comment:
would our IDE be changing it back to 8 spaces?

Reply:
IDE yes, but I do not now why, we should think about solving this issue